src/main/resources/static/js/main.js

```diff
@@ -150,47 +149,65 @@
   const geoCoding = function() {
-    geometry=false;
     var autocomplete = new google.maps.places.Autocomplete(input);
     var infowindow = new google.maps.InfoWindow();
     infowindow.setContent(infowindowContent);
     autocomplete.addListener('place_changed', function() {
-    infowindow.close();
-    var place = autocomplete.getPlace();
-      if (place.geometry) {
+      infowindow.close();
+      geometry=false;
+      var place = autocomplete.getPlace();
+      if (place.geometry) { // User has selected a place from the autocomplete dropdown menu
         geometry=true;
+        var politicalPlaces = [];
         var place_output_str = '';
         if (place.address_components) {
-          //TODO place: just select political types: country - administrative_area (level 1 and 2). Not streets and numbers
-          place_output_str = [
-            (place.address_components[0] && place.address_components[0].long_name || ''),
-            (place.address_components[1] && place.address_components[1].long_name || ''),
-            (place.address_components[2] && place.address_components[2].long_name || '')
-            ].join(' ');
+          for (p in place.address_components) {
+            var politicalFlag = false;
+            for (t in place.address_components[p].types){
+               if (place.address_components[p].types[t] == politicalBoundary){
+                  politicalFlag = true;
+                  break;
+                }
+              }
+            if (politicalFlag && place.address_components[p]){
+                politicalPlaces.push(place.address_components[p].long_name)
+            }
+          }
+        }
+        // Bee.place formatting: deleting duplicates entries (City = Municipality, ie 'Calle Bami 14, Seville') and separated by comma
+        var uniquePoliticalPlaces = [];
+        $.each(politicalPlaces, function(i, el){
+            if($.inArray(el, uniquePoliticalPlaces) === -1) uniquePoliticalPlaces.push(el);
+        });
+        for (s in uniquePoliticalPlaces) {
+           if (s == uniquePoliticalPlaces.length - 1){
+              place_output_str = place_output_str.concat(uniquePoliticalPlaces[s])
+           } else {
+              place_output_str = place_output_str.concat(uniquePoliticalPlaces[s] + ", ")
+           }
         }
-        console.log("place_output :" + place_output_str)
         // Populating field
         form.form('set values', {
-        latitude: place.geometry.location.lat(),
-        longitude: place.geometry.location.lng(),
-        //TODO place_output_str: >>> Bee.place
-        place_output: place_output_str
+          latitude: place.geometry.location.lat(),
+          longitude: place.geometry.location.lng(),
+          //TODO place_output_str: >>> Bee.place
+          //place_output: place_output_str
         });
+        console.log("place_output :" + place_output_str)
       }
     });
   }
-})();
+})();
```

Comment:
I'd prefer to keep an empty line on the end of the file.

Reply:
done

Reply:
nope, wrong line ;). empty line after line 213 not on 212